multiqc/modules/stacks/stacks.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse Stacks 2 denovo output"""
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re
+import os
+from multiqc import config
+from multiqc.plots import table, linegraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+        super(MultiqcModule, self).__init__(name='Stacks', anchor='stacks',
+        href="http://catchenlab.life.illinois.edu/stacks/",
+        info="A software for analyzing restriction enzyme-based data (e.g. RAD-seq).")
+
+        self.gsheaders = OrderedDict()
+        self.gsheaders['n_loci'] = {
+                'title': '# loci',
+                'description': 'Number of loci built',
+                'format': '{:,.i}',
+                'scale': 'RdYlGn'
+        }
+        self.gsheaders['n_used_fw_reads'] = {
+                'title': 'K reads used',
```

Comment:
Instead of this, I wonder if it's better to use the built-in functionality to handle number multipliers: https://multiqc.info/docs/#number-base-multiplier

Did you have a reason not to use this?

Reply:
No, it's deliberately report-wide instead of module specific. You could make a new module-specific config option to make this adjustable, but if you think that `K reads` will basically always give numbers in a sensible range then I guess it's fine to hardcode it...